change_authors_to_dict.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
Are you wanting this to be in the root directory?  I can see these being useful tools for the future, but having the functionality in the notebook in the root directory does not seem right.  I would actually propose making `plasmapy_helpers` package in the root directory that is ignored by `find_packages()` so it's not distributed with `plamsapy`.  We can stuff `plasmapy_helpers` with all the useful pkg management tools we can imagine.

For now, I suggest throwing this notebook into a root directory called `notebooks_for_proto` since that directory is already listed as ignored by `.gitignore`.

Reply:
Oops!  I meant to commit it temporarily so that it was in version control history for future me, but I had thought that I had deleted it.  

But with that said, I do very much like the idea of having something akin to a `plasmapy_helpers` directory that could include things like this.  In the past I had wondered about a separate repo for things like this.

Reply:
Correction: it looks like I already did delete that notebook.